src/fusion_power_plant.cc

```diff
@@ -52,8 +57,8 @@ void FusionPowerPlant::EnterNotify() {
   fuel_startup_policy
     .Init(this, &tritium_storage, std::string("Tritium Storage"),
           &fuel_tracker, std::string("ss"),
-          startup_inventory,
-          startup_inventory)
+          startup_inventory * (1 + trituim_overbuy_multiplier),
```

Comment:
Should we also apply this correction to the reserve inventory?  Or instead of introducing this at all, could we just expect the user to specify a larger reserve inventory to account for this?